spec/system/reviewables_spec.rb

```diff
@@ -54,6 +54,21 @@
         expect(composer).to be_opened
         expect(composer.composer_input.value).to eq(post.raw)
       end
+
+      it "should open a modal when suspending a user" do
+        visit("/review")
+
+        select_kit =
+          PageObjects::Components::SelectKit.new(".dropdown-select-box.post-agree-and-hide")
+        select_kit.expand
+
+        find("[data-value='post-agree_and_suspend']").click
```

Comment:
```suggestion
        select_kit.select_row_by_value('post-agree_and_suspend)
```

I think this should work too.

Reply:
It does indeed! 😀